.github/workflows/examples-esp32.yaml

```diff
@@ -84,3 +85,24 @@ jobs:
 #             - name: Perform CodeQL Analysis
 #               if: ${{ github.event_name == 'push' }}
 #               uses: github/codeql-action/analyze@v1
+            - name: Download Base Binaries
+              uses: dawidd6/action-download-artifact@v2
+              if: ${{ github.event_name == 'pull_request' }}
+              with:
+                  workflow: examples-esp32.yaml
+                  name: esp32-example-build-${{ github.event.pull_request.base.sha }}
+                  commit: ${{ github.event.pull_request.base.sha }}
+                  path: /tmp/base_binaries
+                  repo: project-chip/connectedhomeip
+            - name: Report symbol diff
+              if: ${{ github.event_name == 'pull_request' }}
+              run: |
+                pip3 install cxxfilt pyelftools humanfriendly pandas && \
```

Comment:
should or could we install these packages in the docker image so that we do not run pip install on every commit?

We are trying to make CI faster. If possible, please create a separate PR that updates dockerfile and version.

Reply:
I updated scripts/requirements.in for this yesterday (#6975) so they should get in to future rolls.